plugin/mutate/data/dashboard.html

```diff
@@ -12,6 +12,16 @@
   </head>
 
   <body>
+    <div id ="csp_error">
+      <h2>Something isn't right...</h2>
+      <p>To be able to read the mutation testing report you have to turn off CSP in Firefox.</p>
```

Comment:
If you are using Firefox there may be a problem with Content Security Protection (CSP) with the web server that is used to publish the mutation report. To see the report correctly you need to turn off CSP. Follow these steps: